<launch>
  <arg
    name="model" />
  <arg
    name="gui"
    default="False" />
  <param
    name="robot_description"
    textfile="$(find uav300_head)/urdf/uav300_head.urdf" />
  <param
    name="use_gui"
    value="$(arg gui)" />
  <node
    name="joint_state_publisher"
    pkg="joint_state_publisher"
    type="joint_state_publisher" />
  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="state_publisher" />
  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-d $(find uav300_head)/urdf.rviz" />
  <node
    name="multi_thread_client_gg"
    pkg="wifi_transmitter"
    type="multi_thread_client_gg"
    output="screen"/>
  <node
    name="broadcast_and_show"
    pkg="uav300_head"
    type="broadcast_and_show"
    output="screen"/>

</launch>
